clevertap-core/src/main/java/com/clevertap/android/sdk/DeviceInfo.java

```diff
@@ -105,16 +102,72 @@ private class DeviceCachedInfo {
             bluetoothVersion = getBluetoothVersion();
             countryCode = getCountryCode();
             sdkVersion = getSdkVersion();
-            height = getHeight();
-            heightPixels = getHeightPixels();
-            width = getWidth();
-            widthPixels = getWidthPixels();
-            dpi = getDPI();
+
+            WindowSize result = getWindowSizeData();
+
+            width = toTwoPlaces(result.doubleWidth);
```

Comment:
toTwoPlaces is called twice. From here and while returning from inside `getWindowSizeData()`. Is this intentional?

Reply:
+1

Reply:
fixed this, thanks.